mint/mint.go

```diff
@@ -122,7 +140,7 @@ func (m *Mint) MintLSAT(ctx context.Context,
 	)
 	if err != nil {
 		// Attempt to revoke the secret to save space.
-		_ = m.cfg.Secrets.RevokeSecret(ctx, idHash)
+		_ = cleanup(ctx)
```

Comment:
If the error is never checked, it can just be removed from the returned function.